stock_picking_invoice_link/views/stock_view.xml

```diff
@@ -13,12 +13,13 @@
                         attrs="{'invisible': [('invoice_ids', '=', [])]}"
                         groups="base.group_user"/>
             </field>
-            <xpath expr="//group/field[@name='picking_type_id']" position="after">
-                <field name="invoice_ids" string="Invoice"
-                    groups="account.group_account_invoice"
-                    attrs="{'invisible':[('invoice_ids', '=', False)]}"
-                    widget="many2many_tags"
-                    context="{'tree_view_ref': 'account.invoice_tree', 'form_view_ref': 'account.invoice_form'}"/>
+            <xpath expr="//notebook" position="inside">
+                <page name="invoice" string="Invoices" attrs="{'invisible':[('invoice_ids', '=', [])]}">
+                    <field name="invoice_ids" string="Invoice"
+                        groups="account.group_account_invoice"
+                        widget="one2many_list"
+                        context="{'tree_view_ref': 'account.invoice_tree', 'form_view_ref': 'account.invoice_form'}"/>
```

Comment:
This is not valid when `stock_picking_link_purchase` finally arrives.

Reply:
And why putting this on an extra tab and invoice lines in the same form?

Reply:
> This is not valid when stock_picking_link_purchase finally arrives.
Ohh yea...
I thin that I can remove this and make a new smart button... but when we will have two invoice types... What view display??

> And why putting this on an extra tab and invoice lines in the same form?
I do not understand

Reply:
Yeah, a smart-button with a method that checks linked invoice type and return the corresponding form/tree is a good option.